tests/install.sh

```diff
@@ -6,6 +6,8 @@
 #     s = singularity
 #     t = nf-core/tools
 
+SGT_VER=2.4.5
```

Comment:
I can see that you simplified the actual Travis tests, but that way, you'll have to specify the Singularity version here, and not in the tests.
Wouldn't that be more complicated in the end?

Reply:
I figured that it's pretty rare to need to update the singularity version.. How often does that change?

Reply:
(in contrast to the Nextflow version, which changes a lot)

Reply:
Hmm, yeah ok - fairly often I guess... http://singularity.lbl.gov/all-releases

So in this case should we also have a "minimum singularity version" then?